lnd_test.go

```diff
@@ -863,17 +865,47 @@ func testMaxPendingChannels(net *networkHarness, t *harnessTest) {
 	}
 }
 
-type testCase func(net *networkHarness, t *harnessTest)
-
-var testCases = map[string]testCase{
-	"basic funding flow":          testBasicChannelFunding,
-	"channel force closure":       testChannelForceClosure,
-	"channel balance":             testChannelBalance,
-	"single hop invoice":          testSingleHopInvoice,
-	"max pending channel":         testMaxPendingChannels,
-	"multi-hop payments":          testMultiHopPayments,
-	"multiple channel creation":   testBasicChannelCreation,
-	"invoice update subscription": testInvoiceSubscriptions,
+type testCase struct {
+	name string
+	test func(net *networkHarness, t *harnessTest)
+}
+
+// List of test is much better than map of tests, because in list we have the
```

Comment:
I don't think this comment is _strictly_ necessary.

Although I do like this change as it makes our testing execution more deterministic!